packages/widget/src/components/TokenList/TokenListItem.tsx

```diff
@@ -101,7 +102,11 @@ export const TokenListItemButton: React.FC<TokenListItemButtonProps> = ({
       dense
     >
       <ListItemAvatar>
-        <Avatar src={token.logoURI} alt={token.symbol}>
+        <Avatar
+          src={token.logoURI}
+          alt={token.symbol}
+          sx={{ bgcolor: grey[400] }}
```

Comment:
We can't apply a background to all icons here because icons with transparent backgrounds (like wrapped SOL) will end up with an unintended grey background which looks off, see the screenshot for reference.

<img width="455" alt="image" src="https://github.com/user-attachments/assets/311344ee-6333-4b09-8bc5-bdb3d3032c4e" />